### PR TITLE
crypto.randomUUID

### PR DESCRIPTION
**Pull request by romainmenke**
fixes : https://github.com/Financial-Times/polyfill-library/issues/1089

Not entirely sure that the fallback for the oldest browsers is good enough.
Maybe it is in practice as the number of users is lower?

-----------

Could also be best to wait for a second implementation in a major engine.
Currently this is only supported in Chromium

### CODE REVIEW

**Comment by chee** [comment]
this seems, to me, like a case where it would be better to fail than to provide a weak implementation. the draft says "Overwrite all elements of array with cryptographically secure random values" which is just not something we can polyfill.

also we don't normally polyfill drafts.

**Comment by romainmenke** [comment]
I agree on failing instead of having a bad implementation. I will update the code and config.

I personally also prefer using the Blob trick for all browsers even if it is a bit slower. Any thoughts on this?

Holding this till the spec settles is definitely the right way to go. I mainly submitted the PR so that the issue has a counter part :)

File: polyfills/crypto/randomUUID/config.toml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+bb = "*"
+chrome = "11 - 91"
+edge = "12 - 91"
+firefox = "26 - *"
+ios_saf = "6.1 - *"
+ie = "10 - *"
+ie_mob = "10 - *"
+opera = "15 - *"
+safari = "6.1 - *"
+firefox_mob = "26 - *"
```
Comment (romainmenke) [inline polyfills/crypto/randomUUID/config.toml:33]:
```suggestion
firefox = "26 - 94"
ios_saf = "6.1 - *"
ie = "10 - *"
ie_mob = "10 - *"
opera = "15 - *"
safari = "6.1 - *"
firefox_mob = "26 - 94"
```

https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/95

File: polyfills/crypto/randomUUID/config.toml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+#### Dependencies
+dependencies = []
+
+#### Specification Link
+# TODO update spec link when it settles
+spec = "https://wicg.github.io/uuid/"
+
+#### Documentation Link
+docs = "https://developer.mozilla.org/en-US/docs/Web/API/Crypto/getRandomValues"
```
Comment (romainmenke) [inline polyfills/crypto/randomUUID/config.toml:12]:
```suggestion
docs = "https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID"
```

**Pull request closed without merging**